packages/suite/src/components/wallet/WalletLayout/components/CoinjoinAccountDiscoveryProgress/RotatingFacts.tsx

```diff
@@ -16,20 +16,37 @@ const Fact = styled(motion.p)`
     text-align: center;
 `;
 
-const CJ_FACTS: Array<TranslationKey> = [
-    'TR_COINJOIN_FACT_ONE',
-    'TR_COINJOIN_FACT_TWO',
-    'TR_COINJOIN_FACT_THREE',
-    'TR_COINJOIN_FACT_FOUR',
-    'TR_COINJOIN_FACT_FIVE',
-    'TR_COINJOIN_FACT_SIX',
-    'TR_COINJOIN_FACT_SEVEN',
-    'TR_COINJOIN_FACT_EIGHT',
-    'TR_COINJOIN_FACT_NINE',
-    'TR_COINJOIN_FACT_TEN',
+// better type safety than just storing a count
+const FACTS: Array<TranslationKey> = [
```

Comment:
I would give fact some unique key which describes the fact instead of number or at least number the facts it in the notion.

Reply:
I think numbering the facts in notion is better, since it's less restraining in terms of content.

Reply:
Yea okay, number it in notion 👍